src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -95,6 +96,11 @@ private IEnumerable<string> RootDirectories
             get { return Preferences != null ? Preferences.CustomPackageFolders : rootDirectories; }
         }
 
+        internal string CommonPackageDirectory
```

Comment:
please add a summary even though internal.

Reply:
Done and TODO